src/components/unique-token/UniqueTokenImage.js

```diff
@@ -104,15 +99,15 @@ const UniqueTokenImage = ({
               onLoad={onLoad}
               resizeMode={ImgixImage.resizeMode[resizeMode]}
               size={size}
-              source={{ uri: image }}
+              source={{ uri: imageUrl }}
               style={position.coverAsObject}
             />
-            {!loadedImg && lowResUrl && (
+            {!loadedImg && (
               <ImageTile
                 fm="png"
                 playing={false}
                 resizeMode={ImgixImage.resizeMode[resizeMode]}
-                source={{ uri: lowResUrl }}
+                source={{ uri: item.lowResUrl }}
```

Comment:
@jxom pointing out i removed this based on https://github.com/rainbow-me/rainbow/pull/3188 LMK if I should revert it